nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -212,6 +216,56 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
         temporary_datalayer = self._setup_dataloader_from_config(config=DictConfig(dl_config))
         return temporary_datalayer
 
+    def change_vocabulary(self, new_tokenizer_dir: str, new_tokenizer_type: str):
```

Comment:
Can we introduce a CI test that tests `change_vocabulary()` works with fine-tuning?

Reply:
As in a Jenkins test ? Hmm we'd need to write a new train script for that support. As in the standard speech_to_text_bpe.py won't be able to handle that.

Reply:
There is a unittest that replaces the vocabulary though, if that's what you meant.

Reply:
I saw the unittest which is good, but I feel like we are missing a higher level test that makes sure everything works as intended during fine-tuning

Reply:
Hmm valid point. But then we need scripts for fine-tuning for all ASR + ASR BPE + classification models. What I'm planning is once this PR is done with reviews, to update the speech command notebook to have a section on fine-tuning with change of vocabulary. But that won't be run by CI which is a problem. 

Let's discuss a API spec for fine-tuning with the team and then add it to the ci tests